packages/client/src/create-document.tsx

```diff
@@ -1,42 +1,27 @@
 import React from "react";
-import styled from "@emotion/styled";
 import { useDispatch, useSelector } from "react-redux";
 import { selectUser } from "./redux/appState/user";
 import { addDocument } from "./redux/appState/document";
-
-const Form = styled.form`
-  max-width: 300px;
-  display: flex;
-  flex-direction: column;
-`;
+import { usePostDocumentMutation } from "./redux/api";
 
 export default function CreateDocument() {
   const me = useSelector(selectUser);
-  const [posting, setPosting] = React.useState(null);
+  const [postDocument, { isLoading }] = usePostDocumentMutation();
   const dispatch = useDispatch();
 
+  const handleCreate = async () => {
+    const doc = await postDocument().unwrap();
```

Comment:
what does `unwrap` do?

Reply:
The returned object from postDocument is an object `UserDocument | Error` or something like that, and unwrap() converts this object to just a UserDocument